CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+CHANGES IN VERSION 1.10.0 (from 1.9.0)
+===================================
+
+Appium 1.10.0 is a minor release.
+
+#### General
+* Bring Espresso Driver out of beta and into general availability
+* Support Xcode 10, discontinue support for Xcode 8
+* Improve string extractions for Android and iOS
+
+#### iOS
+* Add biometrics (touchId, faceId) [methods](http://appium.io/docs/en/commands/mobile-command/#ios-xcuitest-only) that can enroll/unenroll biometrics and send matching/non-matching biometric inputs
```

Comment:
added a possibility to upload files into different container types on Simulator: 
https://github.com/appium/appium-xcuitest-driver/commit/5e1ae26644dbb060c3b8594b2f4ac6bea03bc7fa